src/lazyfree.c

```diff
@@ -214,7 +214,8 @@ void emptyDbAsync(redisDb *db) {
 }
 
 /* Release the radix tree mapping Redis Cluster keys to slots.
- * If the rax is huge enough, free it in async way. */
+ * If the rax is huge enough, free it in async way.
+ * Because this rax has only keys and no values so we use numnodes. */
```

Comment:
IMHO these implementation details should be next to the line they refer to, and not in the top comment (that explains what the function does)

Reply:
oh.. i will change it later